apps/fluent-tester/src/E2E/common/BasePage.ts

```diff
@@ -1,13 +1,41 @@
-import { Keys } from './consts';
+import { Keys, ROOT_VIEW } from './consts';
 import { TESTPAGE_BUTTONS_SCROLLVIEWER } from '../../TestComponents/Common/consts';
 import { Attribute } from './consts';
 
 const DUMMY_CHAR = '';
+// The E2ETEST_PLATFORM environment variable should be set in the beforeSession hook in the wdio.conf file for the respective platform
+const PLATFORM = process.env['E2ETEST_PLATFORM'];
+if (PLATFORM) {
+  console.log(`Using platform: ${PLATFORM}`);
+}
 export const COMPONENT_SCROLL_COORDINATES = { x: -0, y: -100 }; // These are the offsets. Y is negative because we want the touch to move up (and thus it scrolls down)
 
+let _rootView: WebdriverIO.Element | null = null;
+
 /* Win32/UWP-Specific Selector. We use this to get elements on the test page */
-export function By(identifier: string) {
-  return $('~' + identifier);
+export async function By(identifier: string) {
+  switch (PLATFORM) {
+    case MobilePlatform.iOS:
+    case NativePlatform.Win32:
+      return await QueryWithChaining(identifier);
+    default:
+      return await $('~' + identifier);
+  }
+}
+
+async function QueryWithChaining(identifier) {
+  if (_rootView === null) {
```

Comment:
Any reason why this variable is prefixed with an underscore instead of just `rootView`?

Reply:
I initially underscored this variable since it's not exported out of the file, though I do see why it's not needed.

Reply:
Renamed.